packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -323,6 +323,16 @@ abstract class LocalHistoryRoute<T> extends Route<T> {
   ///
   /// The given local history entry must not already be part of another local
   /// history route.
+  ///
+  /// ```dart
```

Comment:
When adding sample code, use a `## Sample code` header, so that it triggers this:
https://github.com/flutter/flutter/blob/master/dev/bots/analyze-sample-code.dart

Reply:
Done. Also added an explanation of the example.